### DISCUSSION TITLE
Select All on CRUD Application

### DISCUSSION THREAD

**Discussion by MikeCraig418** [TiagoSilvaPereira/vemto-issues#185] [Ideas & Suggestions]
It would be nice to `Select All` here.

![image](https://user-images.githubusercontent.com/23151962/124036330-b2acb080-d9b2-11eb-971e-c91718e18ea8.png)

**Comment by TiagoSilvaPereira** [comment]
Hi @949mac thank you for your suggestion

**Reply by HRR1337** [reply]
+1
